src/components/ScreenFlatList/header/styled.ts

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components/native';
+import Button from '../../Button';
+import { COLOR_BRAND, COLOR_DARK_1, COLOR_WHITE } from '../../../styles/colors';
+import ChevronBack from '../../Icons/chevronBack';
+
+export const HeaderAreaView = styled.SafeAreaView`
+    background-color: ${COLOR_WHITE};
```

Comment:
here we need to use the attachThemeAttrs and get colors from theme, also change this for the flat list